### PR TITLE
Fix broken tests

### LINKED ISSUE #333: [test] Fix broken tests from the plan command
Some tests have been broken when implementing the `tdp plan` command (#321) as the classes `DeploymentPlan`, `Iterator`, etc have been altered. We need to fix them to cover the new implementations.

### PR DESCRIPTION
**Pull request by sergkudinov**
#### Which issue(s) this PR fixes

Fixes #333

#### Agreements

- [x] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Comment by sergkudinov** [comment]
in `tdp/core/deployment/test_deployment_runner.py`, 3 tests are not currently passing because of #337:
- `test_failed_operation_stops`
- `test_deployment_is_resumed`
- `test_deployment_reconfigure_is_resumed`

**Review by PaulFarault — CHANGES_REQUESTED** [review]
LGTM!

Make sure to remove the `Fixes` from your comment so it won't close the issue (we still have to fix the code and the remaining tests).

**Comment by PaulFarault** [comment]
@sergkudinov Do you prefer to merge this one "now" and do a second pr later to fix the remaining tests or do you want to wait for the #337 to be fixed?

File: tdp/core/deployment/deployment_plan.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ class DeploymentPlan:
 
     Attributes:
         operations: List of operations to perform.
-        deployment_args: Deployment arguments.
+        deployment_log: Deployment information
```
[the remark this answers is not in the document]
Comment (sergkudinov) [reply tdp/core/deployment/deployment_plan.py]: ok

**Comment by sergkudinov** [comment]
I checked this changes on #344 PR's branch, it works. Just waiting #334 merged to master...

**Comment by PaulFarault** [comment]
#344 just got merged, you can go ahead.

**Review by PaulFarault — APPROVED** [review]
#344 just got merged, you can go ahead.

**Comment by sergkudinov** [comment]
@PaulFarault
it works on my computer
43 of 43 passed

**Comment by sergkudinov** [comment]
just rebased on latest master

**Review by PaulFarault — APPROVED** [review]
Ok it works now.

**Pull request merged**